addons/mail/wizard/invite_view.xml

```diff
@@ -14,6 +14,7 @@
                         <field name="partner_ids" widget="many2many_tags_email"
                                 placeholder="Add contacts to notify..."
                                 context="{'force_email':True, 'show_email':True}"
+                                domain="[('type', '!=', 'private')]"
```

Comment:
Shouldn't the domain be directly on the field definition itself in python?

Reply:
Yes. I will move it there.